Doc/cmus.txt

```diff
@@ -12,33 +12,33 @@ cmus [*options*]
 
 @h1 DESCRIPTION
 
-cmus is a small ncurses based music player.  It supports various output
-methods by output-plugins. cmus has completely configurable keybindings and
-can be controlled from the outside via *cmus-remote*(1).
+cmus is a lightweight ncurses music player. It supports various output methods
+by using dynamically-loaded output plugins. cmus has configurable keybindings
+and can be controlled externally using *cmus-remote*(1).
```

Comment:
If anyone has better suggestions for this, I'd be happy to hear them. I only fixed the sentence structure and wording of the previous description, which I don't really like to begin with.